Jenkinsfile_CNP

```diff
@@ -8,6 +8,19 @@ def component = "consumer"
 
 def channel = '#rpe-build-notices'
 
+List<LinkedHashMap<String, Object>> secrets = [
+  secret('send-letter-queue-listen-connection-string', 'SERVICE_BUS_CONNECTION_STRING')
+]
+
+static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
```

Comment:
I am not sure but if using something like `String, Object` will throw Serializable exception as Object is not Serializable?

Reply:
I had a look at CMC code and its using the same so looks like it works :)